eservice/bin/register-with-ledger.sh

```diff
@@ -90,15 +90,16 @@ function Register {
         VAR_BASENAME=$(grep -o 'BASENAME:.*' ${eservice_enclave_info_file} | cut -f2- -d:)
 
         : "${PDO_LEDGER_URL:?Registration failed! PDO_LEDGER_URL environment variable not set}"
-        : "${PDO_LEDGER_KEY_SKF:?Registration failed! PDO_LEDGER_KEY_SKF environment variable not set}"
         : "PDO_IAS_KEY_PEM" "${PDO_IAS_KEY_PEM:?Registration failed! PDO_IAS_KEY_PEM environment variable not set}"
 
-        try ${SRCDIR}/sawtooth/bin/pdo-cli set-setting --keyfile $PDO_LEDGER_KEY_SKF --url $PDO_LEDGER_URL \
-            pdo.test.registry.measurements ${VAR_MRENCLAVE}
-        try ${SRCDIR}/sawtooth/bin/pdo-cli set-setting --keyfile $PDO_LEDGER_KEY_SKF --url $PDO_LEDGER_URL \
-            pdo.test.registry.basenames ${VAR_BASENAME}
-        try ${SRCDIR}/sawtooth/bin/pdo-cli set-setting --keyfile $PDO_LEDGER_KEY_SKF --url $PDO_LEDGER_URL \
-            pdo.test.registry.public_key "$(cat $PDO_IAS_KEY_PEM)"
+        yell WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING WARNING
```

Comment:
We can't register enclaves with the ledger?

Reply:
No, the equivalent of what we had in Sawtooth is not implemented in CCF.

Let me clarify that this isn't the PDO enclave registration transaction.
Rather this is the _trusted_ registration on the ledger of the reference values for verifying PDO enclave registrations later.
The reference values include MREnclave, basename and IAS certificates.
This is a one-time step that we have in the (Sawtooth) CI, after compiling all components (i.e., when MRenclave is available), and before the PDO TP starts accepting enclave registrations.

This is not implemented in the CCF TP. This means that the TP does not have the information necessary to verify the `proof_data` field. For this reason, the TP rejects enclave registrations with non-empty proof-data -- i.e., enclaves compiled in HW mode, while it accepts enclaves compiled in SIM mode, which result in an empty proof-data field.

Reply:
Not sure what you mean..
* the file is about "registration"
* it contains all the relevant keywords like MRENCLAVE, IAS keys, etc.
* it contains the warnings, which show up in the CI
* it clearly states that the registration is skipped (and that is printed in the CI and only triggered in HW mode)

Please add a suggestion for improvements.

Reply:
Added.

Reply:
Committed 039e41